build.sh

```diff
@@ -229,7 +229,7 @@ else
     rm -rf ./'Release-build'
     mkdir ./'Release-build'
     cd ./'Release-build'
-    cmake ../
+    cmake -D CMAKE_BUILD_TYPE:STRING=Release ../
```

Comment:
This option will strip off the assertions in the program. The previous build still keeps the assertions. I am not sure if there is any other option to save the asserts under the release build?

Reply:
Sure, that's not a problem. I added an option to handle that case, which is explicitly enabled for the release build created with the `build.sh` script.